### PR TITLE
[VEN-1478] [VEN-2346] Unlist Market and Fix Borrow Cap 0 Logic

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

**Review by chechu — COMMENTED** [review]
Could we add `checkIsolatedPoolsComptrollers` to the simulations where the IL Comptroller is upgraded?

File: simulations/vip-357/utils/cut-params-mainnet.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        "0xa76b3fda",
+        "0x929fe9a1",
+        "0xc2998238",
+        "0xede4edd0",
+        "0xb0772d0b",
+        "0xabfceffc",
+        "0x007e3dd2",
+        "0xc488847b",
+        "0xa78dc775",
+        "0xddbf54fd"
```
Comment (chechu) [inline simulations/vip-357/utils/cut-params-mainnet.json:16]: I miss `"0xc5b4db55"` (`venusInitialIndex()`), that was added in the [VIP-181](https://app.venus.io/#/governance/proposal/181?chainId=56)

File: simulations/vip-357/utils/cut-params-testnet.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        "0xa76b3fda",
+        "0x929fe9a1",
+        "0xc2998238",
+        "0xede4edd0",
+        "0xb0772d0b",
+        "0xabfceffc",
+        "0x007e3dd2",
+        "0xc488847b",
+        "0xa78dc775",
+        "0xddbf54fd"
```
Comment (chechu) [inline simulations/vip-357/utils/cut-params-testnet.json:16]: I miss "0xc5b4db55" (venusInitialIndex()), that was added in the [VIP-181](https://app.venus.io/#/governance/proposal/181?chainId=56)

**Review by Debugger022 — APPROVED** [review]
Resolve Conflicts and other than that LGTM :)

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/361?chainId=56

**Pull request merged**